<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>390</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Инспектор Аэропортов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="60,40" columnstretch="70,30">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QGroupBox" name="grB_selectZone">
        <property name="title">
         <string>Выбор аэропорта и направления</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QComboBox" name="cbox_listAirports"/>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_in">
           <property name="text">
            <string>Прилет</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_out">
           <property name="text">
            <string>Вылет</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateE_date">
           <property name="maximumDateTime">
            <datetime>
             <hour>23</hour>
             <minute>59</minute>
             <second>59</second>
             <year>2017</year>
             <month>9</month>
             <day>14</day>
            </datetime>
           </property>
           <property name="minimumDateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2016</year>
             <month>8</month>
             <day>15</day>
            </datetime>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::DaySection</enum>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="grB_buttons">
        <property name="title">
         <string>Управление</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pb_getFlight">
           <property name="text">
            <string>Получить список рейсов</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_exit">
           <property name="text">
            <string>Выход</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QGroupBox" name="grB_flights">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableView" name="tv_flights"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="settings"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="file"/>
   <addaction name="help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="settings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
